### DISCUSSION TITLE
posibility to support 2-footprint parts ??

### DISCUSSION THREAD

**Discussion by Vatorinator** [matthewlai/JLCKicadTools#103] [General]
The footprint polulated by Raspberry (contained in the cmio board files) contains 2 times  DF40C-100DS-0.4V (LCSC C597931).
Actually i add manually in the BOM/CPL (delta between the 2 Parts is 39,9 mm) this works
Question: is there the possibility to "automate" this via JLCKicadTools  ???
I attach a picture of the footprint
![RPI4-CM-Module-Footprint](https://github.com/matthewlai/JLCKicadTools/assets/89585863/f713c903-0860-4634-bcd5-d9fd02280897)

**Comment by matthewlai** [comment]
I don't think that's possible in KiCad's architecture unfortunately. Each symbol is associated with one footprint, and each footprint is exported as one placement entry I believe.

To do this within JLCKicadTools one option would be to add another custom field that allows specifying additional part placements in some way. It's not easy though, and I'm not likely to have time to implement that in the near future. Contributions are of course welcome.

Personally I would just add them as two symbols+footprints. One of the footprints can have the board outline and some kind of an alignment marking to help aligning the second connector.

**Comment by Vatorinator** [comment]
thanx a lot for commenting this and i will try out your suggestion.
sorry python is not my world so i can't contribute 

so i mark this as "closed"
